### PR TITLE
DataHandler to create a Dataset from labelled data

### PR DESCRIPTION
**Pull request by sophie22**
Minimalist class that currently handles getting data from Doccano, preprocessing (assuming single label binary task) and tokenisation. Returns a split train and test tokenised dataset with text, label, input_ids and attention_mask.

`torch_base_classes.py` is an example of various training and datamodule arguments for performing PyTorch flavoured model training.

### CODE REVIEW

**Comment by drjzhn** [comment]
@sophie22 @teddyhla 
Thanks Sophie for great work on DataHandler. I've merged in my HFUtils and made the following additions:
(1) DataHandler and HFutils are now merged into hfpipeline.py which abstracts the entire end to end pipeline
(2) Because configurables are shared between different parts of the pipeline (e.g. tokenizer), I've broken out into a shared configuration object as a data class
(3) I've added multi-label preprocessing now to datahandler, that is compatible with the trainer class
(4) You can now specify a configuration object, and run the entire training pipeline in a three lines of code.
(5) I've added a notebook and some sample (pre-labelled) data for the different tasks. Please test this by importing the data into your local Doccano and running the notebook (make sure you set the configuration for the right task).

Other minor change: Now reads the jsonl file name in the doccano zip and uses that. For some reason I was sometimes getting admin.jsonl and sometimes doccanoadmin.jsonl

Next steps
(1) Please test and merge if happy
**(2) Need to work out best way of integrating with MLFlow** (@sophie22)

**Review by drjzhn — APPROVED** [review]
Merge when you have tested and are happy
